### DISCUSSION TITLE
Reporting of successfully ran tests

### DISCUSSION THREAD

**Discussion by ZigaJavornik** [tcunit/TcUnit#244] [Ideas]
Is there a chance you could also report which test was ran (name) when it succeeded? Currently only failed tests are reported and number of tests found. It would be a nice addition imho.

**Comment by sagatowski** [comment]
This is already implemented. You have this information in the info events.

**Reply by ZigaJavornik** [reply]
I didn't have info opened, thanks for the feedback!
